docs/apm/service-overview.asciidoc

```diff
@@ -2,4 +2,106 @@
 [[service-overview]]
 === Service overview
 
-Selecting a <<services,*service*>> brings you to the *Service overview*.
\ No newline at end of file
+The *Service overview* page contains a wide variety of charts and tables that provide
+visibility into how a service performs across your infrastructure. To help you isolate
+potential issues, select a <<services,*service*>> to bring you to the *Service overview* page.
+
+[discrete]
+[[service-latency]]
+=== Latency
+
+Response times for the service. You can filter the *Latency* chart to display the average,
+95th, or 99th percentile latency times for the service.
+
+[role="screenshot"]
+image::apm/images/latency.png[Service latency]
+
+[discrete]
+[[service-traffic-transactions]]
+=== Traffic and transactions
+
+The *Traffic* chart visualizes the average number of transactions per minute for the selected service.
+
+The *Transactions* table displays a list of _transaction groups_ for the
+selected service and includes the latency, traffic, error rate, and the impact for each transaction.
+Transactions that share the same name are grouped, and only one entry is displayed for each group.
+
+By default, transaction groups are sorted by _Impact_ to show the most used and slowest endpoints in your
+service. If there is a particular endpoint you are worried about, click *View transactions* to view the
+<<transactions, transactions overview>>.
+
+[role="screenshot"]
+image::apm/images/traffic-transactions.png[Traffic and transactions]
+
+[discrete]
+[[service-error-rates]]
+=== Error rate and errors
+
+The *Error rate* chart displays the average error rates relating to the service, within a specific time range.
```

Comment:
I think this is fine but just want to clarify that "Error rate" is (somewhat confusingly) based on transaction instead of error documents.
To be exact it is based on the `event.outcome` field on transaction fields and is the relative number of failed transactions. 

We might want to let users in on this secret - or maybe not. Depends on how much detail we can give them without confusing them.

Reply:
I feel like I've seen this come up in discuss in the past. I think it makes sense to keep it high-level here, but maybe update the Error rate info on [this page](https://www.elastic.co/guide/en/kibana/7.11/transactions.html) with the secret sauce.

Reply:
@bmorelli25 Thanks, will do. I included the other update in this PR.